src/tutorials/internationalization.md

```diff
@@ -428,5 +429,24 @@ The DemoLocalizations class uses the generated `initializeMessages()`
 function (defined in `intl_messages_all.dart`) to load the localized messages
 and `Intl.message()` to look them up.
 </li>
-
 </ol>
+
+<a name="ios-specifics"></a>
+### Appendix: Updating the iOS app bundle
+
+iOS applications define key application metadata, including supported locales,
+in an `Info.plist` file that is built into the application bundle. To configure
+the locales supported by your app, you'll need to edit this file.
+
```

Comment:
This list should be sync'd up with Flutter's supportedLocales list: https://github.com/flutter/website/blob/fd92e907636713ee74177f32e0648e11fe2fdd3b/src/tutorials/internationalization.md#specifying-the-apps-supportedlocales-parameter

Reply:
Something like this?

This list should be limited to languages included in Flutter's [supportedLocales list](https://docs.flutter.io/flutter/material/MaterialApp/supportedLocales.html).